.env.example

```
@@ -1,5 +1,9 @@
 DEBUG=True
+POSTGRES_USER=your_postgres_user
+POSTGRES_PASSWORD=your_postgres_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)